storefront/app/globals.css

```
@@ -1,6 +1,23 @@
 @theme {
+  /* ONE serif, ONE sans - the type-system unification.
+     Two serif systems used to coexist: the Tailwind `font-serif` utility resolved to
+     Tailwind v4's DEFAULT stack (ui-serif/Georgia) because no --font-serif override existed,
+     while ~20 bespoke CSS classes (checkout, auth, account, cart drawer) used
+     var(--font-display) ("Iowan Old Style"...). Every wordmark and heading was set in a
+     different face from the forms and drawers - two products in one app.
+     These two declarations are now the CANONICAL stacks. The legacy :root aliases
+     (--font-display, --font-body) are defined as var() references to these, same
+     single-definition discipline as --color-border-control above: bespoke CSS and Tailwind
+     utilities can no longer drift apart, because there is exactly one definition of each. */
+  --font-serif: "Iowan Old Style", "Palatino Linotype", "Book Antiqua", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)